vm.js

```diff
@@ -3273,318 +3662,23 @@ Object.subclass('Squeak.Primitives',
         }
         return false;
     },
+
+    getPrimitiveFunc: function(primIndex, argCount, primMethod) {
+        if ((primIndex > 264) && (primIndex < 520)) {
+            return this.primitiveFunctions[260];
```

Comment:
This surely must be `this.primitiveFunctions[264]`?
Is it possible this is never called? Did you actually change `tryPrimitive()`, which is what the rest of the code is using to invoke a primitive?